src/phoenix/server/api/mutations/chat_mutations.py

```diff
@@ -103,16 +112,16 @@ async def chat_completion(
         )
 
         text_content = ""
-        tool_calls = []
+        tool_calls: list[ChatCompletionToolCall] = []
         events = []
         attributes.update(
             chain(
-                _llm_span_kind(),
-                _llm_model_name(input.model.name),
-                _llm_tools(input.tools or []),
-                _llm_input_messages(messages),
-                _llm_invocation_parameters(invocation_parameters),
-                _input_value_and_mime_type(input),
+                llm_span_kind(),
+                llm_model_name(input.model.name),
+                llm_tools(input.tools or []),
+                llm_input_messages(messages),
+                llm_invocation_parameters(invocation_parameters),
+                input_value_and_mime_type(input),
```

Comment:
these are now deduped and imported from playground span, not because i love deduplication, but because the input should be the same and handled in the same way for the mutation and the subscription. I kept fixing things in both places / changes to the playground span handlers were not propagted here leading to various bugs. Note output handlers is not deduped because the outputs of these are inherently different, if the inputs diverge, we can do the same